src/mlpack/methods/decision_tree/decision_tree_main.cpp

```diff
@@ -39,7 +39,8 @@ PROGRAM_INFO("Decision tree",
     "may not be specified when the " + PRINT_PARAM_STRING("training") + " "
     "parameter is specified.  The " + PRINT_PARAM_STRING("minimum_leaf_size") +
     " parameter specifies the minimum number of training points that must fall"
-    " into each leaf for it to be split.  If " +
+    " into each leaf for it to be split.  The " + PRINT_PARAM_STRING("minimum_gain_split") +
+    " parameter specifies the minimum gain that is needed for the node to split. If " +
```

Comment:
I think it might be useful to add a little intuition here---maybe we should point out that a larger `minimum_gain_split` is a form of regularization?

Reply:
I think, adding this intuition in the header file during function declaration would be better, just like how it is done for other parameters.